filesystem/luks.ksy

```diff
@@ -0,0 +1,72 @@
+meta:
+  id: luks
+  title: Linux Unified Key Setup
+  license: CC0-1.0
+  endian: be
+  encoding: ASCII
+doc: |
+  Linux Unified Key Setup (LUKS) is a format specification for storing disk
+  encryption parameters and up to 8 user keys (which can unlock the master key).
+doc-ref: https://gitlab.com/cryptsetup/cryptsetup/wikis/LUKS-standard/on-disk-format.pdf
+seq:
+  - id: partition_header
+    type: partition_header
+types:
+  partition_header:
+    seq:
+      - id: magic
+        contents: [0x4C, 0x55, 0x4B, 0x53, 0xBA, 0xBE]
+      - id: version
+        contents: [0x00, 0x01]
+      - id: cipher_name_specification
```

Comment:
Look like a new type.
```yaml
spec_str:
  seq:
    - id: str
      type: str
      size: 32
```

Reply:
I wouldn't go as far as creating new type for a mere repetition of same 32-byte string for 3 times.